src/Facets/AllBridgeFacet.sol

```diff
@@ -96,14 +126,48 @@ contract AllBridgeFacet is ILiFi, ReentrancyGuard, SwapperV2, Validatable {
         ILiFi.BridgeData memory _bridgeData,
         AllBridgeData calldata _allBridgeData
     ) internal {
+        // we do not validate _allBridgeData.fees here due to gas optimization reasons
+        // our backend ensures that the fees are correct
+
+        // make sure destinationChainId matches in bridgeData and allBridgeData
+        if (
+            _allBridgeData.destinationChainId !=
+            _getAllBridgeChainId(_bridgeData.destinationChainId)
+        ) revert InvalidCallData();
```

Comment:
Why would you keep `_allBridgeData.destinationChainId`? We can remove the parameter and only use the mapped value: `_getAllBridgeChainId(_bridgeData.destinationChainId)`.